tla/Traceetcdraft.cfg

```diff
@@ -64,12 +61,14 @@ CONSTANTS
     InitServer <- TraceInitServer
     Server <- TraceServer
 
+    IsEnabled <- TraceIsEnabled
+    PostAction <- TracePostAction
+
 INVARIANTS
     LogInv
     MoreThanOneLeaderInv
     ElectionSafetyInv
     LogMatchingInv
     QuorumLogInv
     MoreUpToDateCorrectInv
-    LeaderCompletenessInv
-    CommittedIsDurableInv
\ No newline at end of file
+    LeaderCompletenessInv
```

Comment:
Why is `CommittedIsDurableInv` no longer an invariant?

Reply:
The invariant is not correct. Leader does not persist committed entry before PersistState action.